packages/eslint-config-default/README.md

```diff
@@ -0,0 +1,24 @@
+# @wordpress/eslint-config-default
+
+Default [Eslint](https://eslint.org/) config for WordPress development.
```

Comment:
This ESLint configuration is **not** for "WordPress" development, it is for "Gutenberg" development:

> `Default [Eslint](https://eslint.org/) config for Gutenberg development.`